packages/mds-types/transformers/vehicle_event_transformers.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  VEHICLE_EVENT,
+  VehicleEvent_v0_4_1,
+  VEHICLE_REASON_0_4_1,
+  VEHICLE_EVENT_0_4_1,
+  VehicleEvent,
+  VEHICLE_STATE
+} from '../index'
+
+function convert_event_type_and_event_type_reason_to_event_type(
```

Comment:
This method name is really verbose but in a non-meaningful way IMO, if the helpers have versions encoded in the name, this probably ought to as well (additionally this also seems like a helper but doesn't have `helper` in the name like some others do).